tripy/tests/integration/test_batchnorm.py

```diff
@@ -0,0 +1,58 @@
+import torch
```

Comment:
Do you have the pre-commit hook enabled? This file should have had a copyright header added.

Reply:
fixed now, thanks!